docs/docs/guides/deploy/compose-pg.mdx

```diff
@@ -0,0 +1,29 @@
+---
+title: Docker Compose using Postgres
+---
```

Comment:
this will not show up in the guides... do we want to have a separate link in the navigation or just a separate section in the compose page? @adlerhurst or @eliobischof

Reply:
we decided to omit a dedicated compose file